qcodes/instrument_drivers/tektronix/AWG70000A.py

```diff
@@ -146,7 +146,7 @@ def validate(self, value: float, context: str='') -> None:
             validator.validate(value)
 
 
-class AWGChannel(InstrumentChannel):
+class Tektronix70000AWGChannel(InstrumentChannel):
```

Comment:
leaving `AWGChannel = Tektronix70000AWGChannel` alias would avoid the need for reporting the breaking change, right?